packages/logging/src/log.js

```diff
@@ -520,8 +526,6 @@ Log.prototype.warning = function(entry, options, callback) {
  * @param {object[]} options.labels - Labels to set on the log.
  * @param {object} options.resource - A default monitored resource for entries
  *     where one isn't specified.
- * @param {boolean} options.removeCircular - Replace circular references in an
```

Comment:
I'll take your word for it on the use case. If it's not common, then let's keep it out.